tensorflow/lite/experimental/ios/BUILD.apple

```diff
@@ -11,16 +11,18 @@ package(
     licenses = ["notice"],  # Apache 2.0
 )
 
-TFL_FRAMEWORK_HDRS = [
-    "//tensorflow/lite/delegates/gpu:metal_delegate.h",
-    "//tensorflow/lite/c:c_api.h",
-    "//tensorflow/lite/c:common.h",
-]
-
 # bazel build -c opt --config=ios_fat //tensorflow/lite/experimental/ios:TensorFlowLiteC_framework
 ios_static_framework(
     name = "TensorFlowLiteC_framework",
-    hdrs = TFL_FRAMEWORK_HDRS,
+    hdrs = [
+        "//tensorflow/lite/c:c_api.h",
+        "//tensorflow/lite/c:common.h",
+    ] + select({
```

Comment:
I think the plan is to move Metal delegate to its own framework, similar to CoreML delegate:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/ios/BUILD.apple#L67

In which case CoreML and Metal delegates will depend on `TensorFlowLiteC_framework`. So the Metal delegate selects in this target will eventually be moved.

Since this is probably out of scope for your change, could probably leave it as you have it and we can migrate metal delegate to match CoreML delegate in follow-up change.